packages/contracts/contracts/vendor/SignerRole.sol

```diff
@@ -10,8 +12,11 @@ contract SignerRole {
 
     Roles.Role private _signers;
 
-    constructor () internal {
-        _addSigner(msg.sender);
+    Holding parentContract;
+
+    constructor (Holding _parentContract) internal {
```

Comment:
Why using `_parentContract`? If inheritance is used, `this` should be the same as `_parentContract`. No?